ios/Classes/NetIamyellowTiwsWSProxy.m

```diff
@@ -121,8 +124,11 @@ -(void)close:(id)args
 
 -(void)send:(id)msg
 {
-    ENSURE_SINGLE_ARG(msg, NSString);
-
+    ENSURE_SINGLE_ARG(msg, NSObject);
+    if ([msg isKindOfClass:[NSString class]])
+        msg = [TiUtils stringValue:msg];
+	else // TiBlob
```

Comment:
this `else` should be an `else if` for checking if the `msg` argument is a TiBlob, with the final else throwing an exception stating that the argument is of unsupported type